types/apollo-upload-client/index.d.ts

```diff
@@ -1,22 +1,28 @@
-// Type definitions for apollo-upload-client 8.1
+// Type definitions for apollo-upload-client 14.0.0
 // Project: https://github.com/jaydenseric/apollo-upload-client#readme
 // Definitions by: Edward Sammut Alessi <https://github.com/Slessi>
+//                 Jordan Overbye <https://github.com/jordanoverbye>
+//                 Simen Bekkhus <https://github.com/SimenB>
```

Comment:
I'd prefer not - I don't wanna be tagged for review in future changes 🙂

Reply:
Done